src/design-system/components/Drawer/Drawer.tsx

```diff
@@ -0,0 +1,39 @@
+import DrawerProps from './Drawer.types';
+
+import { motion } from 'framer-motion';
```

Comment:
hmmm, it goes a bit beyond the scope of the work here, but maybe we could remove `framer-motion` as a dependency?

I _thiink_ it's only used here and it might reduce our bundle size a bit + remove a bit of an overkill library that just animates the drawer coming in and out.. something we could probably do quite easily with CSS transitions perhaps? :)

Reply:
Yeah, I had wanted to strip it out, but struggled to get the css animation to do what I want. Think I've had an idea for how we might approach it. I'll merge it in and spin up a separate PR for that. 😄

Reply:
sounds good! :)